base_comment_template/models/base_comment_template.py

```diff
@@ -124,38 +102,44 @@ class BaseCommentTemplate(models.Model):
         help="This comment template will be available on this models. "
         "You can see here only models allowed to set the coment template.",
     )
-
     domain = fields.Char(
         "Filter Domain",
         required=True,
         default="[]",
         help="This comment template will be available only for objects "
         "that satisfy the condition",
     )
-
-    priority = fields.Integer(
-        default=10, copy=False, help="the highest priority = the smallest number",
+    sequence = fields.Integer(
+        required=True, default=1, help="The smaller number = The higher priority"
     )
 
-    @api.constrains("domain", "priority", "model_ids", "position")
+    @api.constrains("domain", "sequence", "model_ids", "position")
```

Comment:
Why this constraint? With this new approach, you can add as much templates as you want. The `sequence` only determines the order in which they are going to be applied.

Reply:
Constraint have removed.